docs/api/openapi.yaml

```diff
@@ -44,47 +43,60 @@ info:
     url: ""
 servers:
   - url: https://{server_url}/apis/results.tekton.dev
-    description: The common url format for tekton results API
+    description: The common URL format for Tekton Results API
     variables:
       server_url:
         default: localhost:8080
         description: URL of the API server
-    x-last-modified: 1677677605032
+    x-last-modified: 1691482642845
 paths:
-  /v1alpha2/parents/{parent}/results:
-    summary: List Results
+  /v1alpha2/parents/{log-name}:
```

Comment:
:thinking: I see why it is rendering like this. The protobuf spec for the "name" includes a pattern match that nests pretty deep: https://github.com/tektoncd/results/blob/main/proto/v1alpha2/api.proto#L99. Fixing this to be more REST-ful would break the protobuf API.

We are ultimately arguing because the term "name" is way too overloaded. Are we talking about a Kubernetes object name? A record identifier in the Tekton Results database?

@avinal are you still writing this by hand? Or are you generating the openapi YAML?

Reply:
True, the term `name` is too overloaded, and I don't see a legitimate way of simplifying. @xinnjie, if you can join our next WG call, we can have a discussion and agree on an approach to remove this ambiguity.

>  are you still writing this by hand? Or are you generating the openapi YAML?

Sadly yes, existing solutions don't work great or too complex for our use case. I am thinking of taking inspiration and then implement our own.